base/asyncevent.jl

```diff
@@ -306,7 +306,7 @@ Waits until `testcb()` returns `true` or `timeout` seconds have passed, whicheve
 The test function is polled every `pollint` seconds. The minimum value for `pollint` is 0.001 seconds,
 that is, 1 millisecond.
 
-Returns :ok or :timed_out
+Return :ok or :timed_out
```

Comment:
```suggestion
Return `:ok` or `:timed_out`.
```

Reply:
Perhaps also `Waits` -> `Wait` above while we are here.

Reply:
Aye, there is plenty more to be done :-)